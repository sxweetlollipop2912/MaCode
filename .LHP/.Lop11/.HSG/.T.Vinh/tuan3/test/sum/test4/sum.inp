3000
99947046
99859099
99772972
99650633
99521474
99484071
99397128
99276501
99196030
99029171
98952356
98836897
98795482
98609983
98537728
98469549
98312214
98244203
98103100
98057561
97994130
97835351
97749688
97667077
97593774
97453859
97382516
97292593
97128298
97004079
96971440
96803645
96726630
96618267
96531852
96450441
96340514
96229607
96145288
96044181
95970622
95801811
95729156
95678433
95513338
95465471
95367008
95247565
95177526
95038699
94984668
94894009
94729714
94616823
94516344
94434693
94337134
94223139
94137236
94040561
93975850
93874671
93782864
93662909
93509894
93420347
93398924
93249641
93163362
93005543
92978376
92859637
92718430
92636755
92578628
92491297
92363130
92233055
92124576
92091309
91991030
91809163
91739420
91649017
91521938
91471127
91327256
91275141
91151438
91004035
90951252
90866449
90776362
90669231
90543984
90495389
90356838
90237627
90128332
90084137
89947554
89892935
89703752
89676917
89584030
89435187
89343620
89215329
89151514
89029055
88987392
88814189
88745558
88682059
88544348
88495449
88390258
88229879
88125368
88057573
87955918
87844675
87748436
87605233
87523882
87447407
87325520
87204125
87176774
87018043
86919916
86880105
86790818
86657607
86532648
86412981
86312254
86292179
86161732
86029697
85976282
85889151
85752640
85643469
85566902
85477739
85322140
85245497
85177970
85076759
84922904
84881349
84786414
84687075
84563764
84480273
84319946
84237263
84167472
84073309
83944678
83854363
83768268
83652105
83549314
83417447
83312872
83233237
83113598
83029395
82999908
82836545
82710010
82666495
82521376
82420269
82330166
82226795
82106588
82083385
81903154
81880215
81770552
81626981
81511726
81448899
81328852
81221553
81191818
81000687
80948304
80851933
80768486
80631355
80513676
80454025
80386690
80291463
80196104
80049653
79947870
79811443
79787140
79642753
79516410
79466303
79314880
79234157
79159766
79019115
78977180
78849017
78725970
78693463
78520984
78488837
78333614
78258051
78186420
78063793
77939658
77835279
77799408
77619197
77556390
77444027
77342124
77279081
77141378
77083431
76954248
76822645
76791486
76645875
76579876
76461825
76333978
76278559
76158304
76053965
75929430
75871179
75742748
75637977
75579474
75454487
75381688
75258789
75135854
75036323
74924884
74831985
74722058
74607055
74572400
74484509
74314950
74213979
74150700
74008873
73959298
73816487
73763080
73675541
73570174
73417779
73314852
73212001
73157754
73094367
72929728
72813325
72729494
72609707
72555932
72452473
72338802
72253335
72196728
72026213
71902478
71813859
71727412
71688561
71539690
71499503
71339120
71222077
71150982
71089403
70982188
70847977
70777506
70680295
70551304
70496469
70358974
70287891
70156260
70085441
69975802
69856895
69778400
69627021
69599606
69404363
69349660
69215801
69186578
69064823
68930392
68827045
68783182
68649283
68590164
68453521
68382474
68231823
68158928
68019965
67936966
67845083
67725068
67673001
67591490
67467527
67302440
67203765
67165886
67027187
66930340
66828097
66765434
66694079
66573792
66440045
66328726
66251403
66184316
66057881
65963858
65864887
65767192
65629317
65544270
65409571
65320404
65284305
65133354
65040687
64927760
64863741
64778342
64682395
64562572
64483529
64336930
64288327
64169288
64080565
63931870
63833235
63776324
63659841
63513338
63418623
63354656
63211245
63122422
63057323
62949628
62897017
62735538
62697879
62579160
62447621
62317710
62255587
62187444
62028625
61915466
61814831
61745424
61669181
61513510
61445659
61339980
61283881
61107938
61021255
60908520
60840213
60779902
60646547
60544548
60472577
60346394
60211647
60198208
60096973
59968278
59871563
59730076
59658137
59557650
59460215
59391256
59210405
59168430
59058435
58906292
58822577
58759434
58627759
58548304
58416861
58313830
58205787
58131916
58085001
57989762
57829639
57764328
57626261
57500510
57417875
57363748
57246593
57149914
57022687
56950944
56818381
56754134
56686507
56555836
56454553
56361010
56282135
56115960
56010917
55977966
55881219
55754004
55657937
55537962
55437743
55360304
55217565
55116326
55043931
54924108
54853513
54758786
54628871
54517480
54414357
54382590
54224563
54175652
54052449
53949402
53818751
53763232
53626829
53533494
53488427
53319132
53255257
53183922
53096887
52969624
52888965
52709870
52631875
52586036
52449457
52347594
52271471
52158256
52079581
51974630
51867835
51764332
51660777
51515234
51451815
51385320
51226709
51131646
51003251
50949444
50898145
50796090
50618911
50589312
50420237
50376470
50222891
50106940
50052217
49923122
49887895
49737880
49649157
49527950
49410339
49352596
49200273
49176010
49040399
48973808
48892349
48708038
48644539
48513356
48468425
48301410
48283399
48140008
48015733
47918846
47851763
47767300
47649249
47581402
47494303
47320000
47228621
47135190
47045707
46935580
46809433
46715538
46674295
46593880
46409957
46378030
46210435
46116884
46020145
45985674
45824399
45740592
45691869
45568806
45434971
45375756
45210121
45183458
45093319
44955688
44882293
44731966
44692627
44589796
44492417
44355194
44249887
44117280
44012205
43945686
43839275
43716124
43662937
43523826
43459991
43381944
43203685
43189838
43085283
42920500
42839921
42730186
42688111
42514352
42478941
42325798
42243003
42169100
42007913
41985954
41819687
41728616
41657461
41591742
41476851
41388612
41253633
41191162
41009983
40928768
40845069
40742358
40645291
40588988
40471737
40383538
40233655
40132024
40060165
39907822
39881891
39799796
39680721
39554282
39497295
39336464
39284573
39197190
39099099
38989356
38874121
38757314
38685383
38557032
38486165
38384574
38263795
38115716
38098209
37937786
37849087
37740544
37698797
37559862
37482987
37305692
37250521
37139602
37052375
36961592
36823077
36760334
36673859
36536820
36452049
36326538
36253423
36135824
36004573
35983174
35876667
35794540
35625801
35537090
35430759
35377640
35278133
35129822
35074003
34983236
34847137
34701978
34644703
34524320
34458829
34382166
34230891
34147100
34086009
33977138
33874359
33775640
33600901
33525038
33410435
33375668
33286385
33157610
33042735
32945200
32881309
32784198
32659355
32504716
32419337
32323266
32282823
32112776
32089653
31979806
31836371
31799940
31677089
31547194
31491807
31322976
31275405
31142550
31094123
30972476
30804249
30756050
30626231
30548440
30465029
30345262
30218403
30138836
30020945
29965642
29885295
29750288
29654653
29533670
29441051
29334700
29250441
29137666
29051463
28939592
28841813
28719806
28608275
28552996
28436769
28362618
28253791
28199616
28028973
27923606
27804395
27735836
27686969
27543730
27470327
27341784
27235301
27164590
27083843
26978164
26883217
26776266
26687823
26524304
26429277
26314310
26291291
26128652
26086857
25960546
25833223
25788168
25602261
25500766
25471795
25324324
25281505
25161274
25095167
24937568
24813837
24708182
24620011
24587836
24471577
24366002
24219159
24121336
24061765
23924974
23803075
23772948
23605425
23589898
23423567
23344336
23240061
23190854
23057147
22909132
22835657
22796002
22674823
22508808
22403445
22306590
22270195
22141540
22087713
21947322
21816159
21756736
21621805
21577494
21425867
21322940
21234809
21156178
21020951
20972984
20891973
20752270
20699011
20515476
20442769
20398090
20285135
20140400
20035741
19911942
19877691
19777100
19659593
19522914
19458695
19322984
19208021
19137790
19006035
18977956
18805729
18716058
18634463
18580896
18409485
18396662
18249803
18132252
18041785
17938706
17808695
17719192
17687333
17521646
17437507
17392788
17206129
17199882
17066255
16930032
16862557
16791878
16636827
16595564
16477097
16311298
16252039
16121448
16043445
15960542
15892115
15717988
15606977
15537562
15419839
15321344
15212749
15112822
15097195
14934332
14864249
14745522
14667447
14586296
14490405
14390286
14211011
14137620
14078417
13936394
13845423
13750992
13647901
13594278
13426811
13356876
13278633
13105954
13023175
12998824
12800085
12790334
12600531
12575396
12469505
12305754
12243167
12125952
12083213
11906070
11865419
11760732
11662329
11566994
11432439
11320856
11257605
11131982
11038563
10977460
10881873
10721642
10653199
10561264
10433021
10383014
10201083
10164492
10061289
9975522
9876135
9704552
9600405
9542910
9480915
9342340
9259745
9140282
9011039
8979168
8851757
8773110
8662091
8537756
8451833
8317330
8225559
8156952
8075557
7920782
7833091
7752788
7678065
7562666
7421263
7339056
7289373
7120326
7088731
6983916
6810537
6747074
6632423
6506408
6446133
6332382
6258547
6173028
6083457
5933690
5863871
5752960
5697709
5559958
5488715
5353756
5238841
5152818
5018871
4981592
4862981
4745454
4699523
4531540
4416945
4339402
4249711
4166448
4053213
3968710
3829083
3709996
3651177
3571522
3465607
3394280
3271093
3140414
3085299
2983716
2885729
2704250
2677791
2539520
2430605
2394646
2206091
2159132
2083225
1986290
1836503
1785656
1634213
1595022
1420323
1310996
1271441
1190218
1023407
962096
874653
756326
604411
532332
480041
358210
250375
143688
55669
9463390
48374227
69685412
74567329
18794074
73946399
71051264
52257485
62489110
77101611
92066332
91118777
156306
52687351
82647544
88319333
82375118
83520899
27950484
51586513
43411914
40295631
64232176
84534013
77527942
19590875
36518924
333289
11640322
2929831
79001320
91861397
63681598
80663091
35402116
97617409
84320058
90582655
62820832
46899245
61320950
55305611
46366204
60951321
4341618
15816023
58014936
68411333
46989742
55269603
17204084
36930097
58774186
75220783
26203088
26776413
51261286
38507579
30550764
53202761
21393122
81215495
97734088
93580021
83240990
17367187
51485284
38331745
16706586
14249695
5201088
84257933
81600726
62025963
21795292
99475065
20579410
50117303
33423032
18300709
54726798
39428675
95871572
5374865
29839754
75288975
26154160
14960829
52432454
42432923
40552652
69619625
73515714
34903655
41645480
31684437
6653182
66712051
24083012
49278145
8152826
11230015
27516064
55333101
18804662
3029835
41668540
91290841
34630706
72205335
81894552
66326405
73672558
2436003
57498420
36683761
94027882
75607791
91419536
87960093
22285094
11539195
93429676
45187081
20214690
23175367
13059208
8645045
21513694
43282515
39170852
99996449
36265434
75390879
65175424
46870861
22525590
90443691
51579292
37971513
7598610
31311735
30021496
41606629
20997198
7023363
22941204
85140817
21648970
33836879
50386288
75360893
18761222
27325275
95914380
39964009
41555074
80280103
52770152
70162453
79600062
10473651
99970052
66933121
6509498
85733375
39543648
31298477
48989302
80171531
14366588
5545625
91026418
99220183
42867032
6424389
25536558
17249123
30302196
59420337
58355498
40253359
79978064
92431069
1403110
77649339
24501356
73432265
3009378
68052103
2626120
95102069
86806430
38039827
74431204
1055713
14790298
86261599
73038144
14208013
13570902
81960299
62631516
24497401
11799506
75299127
86547256
61260197
75275790
51014595
82413012
85168657
87723786
2653711
95589680
40119613
41418694
94147099
67898188
34562089
50424898
75910375
47978792
58075861
13208958
92029299
10332356
55205697
32355194
6113471
31580192
1970797
43311158
6883531
51253564
37250393
39759794
86501783
65649432
44792069
64833006
27547683
14449076
10036849
26221290
13968239
61152528
20154781
16648870
67329915
42123564
71746697
15056418
63309639
22716424
19376949
25451358
92333267
20180900
56745633
15859034
96978975
77111296
76573901
19367446
72374059
57390620
50683833
67574930
87366135
85813752
38928229
95459278
14886531
68766100
83549905
75722954
37162959
84101872
9880573
11567494
36519387
32958220
55384041
61049858
27354791
99198440
24979605
11908414
60169523
33729924
92461057
7364154
85616511
15076832
20783661
29530102
92158603
70413564
41165337
25836402
2465879
48700632
16290757
80071342
87428579
67547252
24721201
13445802
15305007
20409552
34903901
69797734
94880059
48461292
57745993
80409570
82670087
3353544
69121013
57522462
65566099
60559972
61076833
76791322
78885599
46942400
50531213
20738774
81290475
12037084
91969145
25510482
41508535
94441656
77633829
33186190
68381507
97705044
22052497
54530698
19046287
17034928
53996477
41595718
79354011
8904908
25625513
8096450
23980903
47065256
21654101
58901502
84910323
22464580
78289857
84501498
11230527
70193824
19945197
99082422
8665931
63672508
50825177
62970930
36753687
64221336
24490885
80986478
29763235
20946740
65919473
94493290
89074415
74506640
69160221
88883238
38275323
61872300
65238537
49380770
21081031
48302728
5727413
71802846
43399763
10999332
76769057
66197722
39647647
25820800
26384461
77282966
44507819
34042268
34951737
14966290
48045687
95244408
44076517
61320526
2467331
16225300
21218897
66773066
3727183
26791792
73110909
75182342
16271963
24708748
47256681
99972994
23865383
10989928
25137941
11133118
20074931
45030916
96089985
42990522
48718079
55865952
5405357
74695286
12881931
96598140
20990105
96168946
57780951
63186776
9286469
8604206
61164899
32255732
64786865
32667946
6345391
61426768
39212765
75680742
11814843
84521580
59323593
88474722
4780935
40103752
74396533
40882846
90270227
60671716
92800225
68881562
50607711
88326720
25729037
22339926
81631595
99586652
2553593
44077010
80973367
49809976
83730341
26468110
86886339
92547796
85398801
86366986
25403663
9451824
89092925
24715462
19668763
45663820
95989545
76378690
63859431
84124968
86211541
46773630
88195699
58763460
92935745
30763642
32550847
57286176
51823213
5354038
77475787
8500028
85194841
39145138
55188887
70314008
21731589
72659438
9472291
85340596
83769457
59983082
11928687
97927440
30059677
63256486
45990523
34766892
73809545
53036066
28717383
35038216
84005173
58578270
91838931
19975332
29439137
48420698
61882911
30265856
38869453
15507734
75943723
11108892
86405817
37170066
45578231
93468152
15811173
59041998
12606339
21151636
30101969
8454858
92015567
97417968
55102461
33874054
35681755
740620
21277929
45721858
54556071
33364712
29397141
70251582
38063155
37189508
52225025
74527802
81007743
28356576
70181421
8753142
58923915
17462268
45683225
84388722
44876887
31545560
49203909
9145262
81298659
32776820
54022449
29677482
24376879
21991632
440925
48353126
50068795
62256876
26312009
62085858
19128839
10613192
74721013
87414046
60025491
78082916
75663713
94651162
54881247
68669376
29852045
42642646
99566059
2338524
9430393
2380114
73405111
92839608
90825765
5648014
1886019
22774100
2129041
49404554
85244559
86769584
87975869
15046214
36706459
55233228
46317481
42668994
27773799
99345320
10507861
17286398
44726003
80094532
26064833
27248122
28560447
59302816
87655405
86394294
82411851
62794428
88506329
50647090
21518871
24114840
47790213
32829294
9515427
58430772
37991921
78797162
35984367
50376848
35354653
11520550
7057915
37865644
50639369
13419682
18446535
23110280
12969909
78755038
33007699
97909028
46710049
71150810
74687135
51859328
46530381
10826390
88296875
70681500
10742073
69067282
64707959
78221432
22957285
39150158
93176835
39377940
79571537
56874826
13030479
9909616
40873085
11910918
16363867
38112140
73078377
80661122
99138343
1477480
54065941
92306622
9491123
61005828
40304001
63115194
40971519
51510624
57678765
55971958
99383307
15096956
15907993
30409458
1014231
46480728
52351813
79704622
95703139
19910644
66833073
13097258
70336431
68549712
98573533
54534118
26224315
58725484
59374537
83607650
5425287
50036552
86403189
95126174
17672467
68626404
31556065
67755674
50192223
1899584
52950797
5915222
56708971
49867356
60746489
87817170
51031607
16234296
10196133
63703310
56221123
39184084
51681041
52266250
52022287
67950128
88179517
84339910
47049499
14671436
24756265
94364738
47951847
32913704
10851797
25998974
2374515
84741572
52082241
35094138
52712639
42689312
59943277
13955126
92572875
8646716
61422937
37390514
20487831
70843928
66459141
12022510
20216867
48566452
2991985
64591082
72243823
45784080
27612061
5427366
90576251
48226860
43874953
17928994
22496327
52706312
61446709
82859614
57230291
633252
97954209
95527258
31513631
62079488
91934157
34990870
45633323
89818396
3591097
93022610
37662711
89691640
24274533
52171470
34503043
69187988
96549073
25688522
67708879
35745008
25506045
28528518
74900699
66397964
3321321
44704514
89840039
60548328
10420373
22791998
24683059
24829060
34732033
17375546
34579071
39257312
52915245
35587318
8391563
14044156
32327705
64079474
53388119
95663320
77489861
18292398
94702563
49490036
30140209
91550122
7742767
67546576
28693853
18492006
61896507
3502060
11690825
40437474
95898119
3593928
73235445
9512990
58568083
83102308
87398753
96817946
42510303
54462912
32559501
83909590
22159083
71747804
9681529
35269202
98597047
60181432
10666533
51160462
97764931
50126932
8459921
51058570
84222863
14406320
74721725
4348486
19796635
6069452
84485545
13830594
56621415
77533864
51035733
13372414
3981811
28537412
45393089
15440890
16009791
26407584
11253741
12304822
77057611
23115196
57124313
29223730
84323607
45655960
41530757
8249198
41966243
54031412
5691121
25987690
21644015
50594704
91849757
21761574
18160635
47941548
54179593
48928674
20578247
64013704
35678901
68902110
59863635
31464484
15125793
82689242
38332063
22339200
12614989
59753110
22084523
98094236
18132537
96433426
91637623
15549816
36071653
91083342
84458243
71447316
60472401
23518794
14536783
36337008
88986493
7995142
80391003
15172748
17702761
67700354
11405351
50764648
9736469
7201470
84028595
26942980
62430593
50964410
67800063
58042208
45941421
19351158
2531083
48911228
95605657
72796146
34226391
71797080
93443141
70402350
73653859
72315124
18348977
26175274
37532847
32911440
75819741
64495078
73667771
26161260
83924169
67456354
22775175
11472712
93977461
28584606
25552915
29859812
70113249
42977178
42837855
40288576
53696013
48377686
65015147
97554524
4382457
79617234
95812919
17384760
50996645
18545934
16841667
6402772
46870801
59437066
87815951
97616432
90169405
99340230
76562971
6485580
26168617
88463938
86010343
73393192
42335701
82449534
92388467
67314884
85435201
76911226
24421567
71870496
79121005
95318
197515
258172
373753
444114
537687
687384
797445
803118
904131
1074836
1127153
1226186
1303663
1454288
1518301
1658758
1726171
1868012
1949289
2016098
2136487
2252552
2391573
2476286
2581011
2619204
2713569
2875962
2944799
3016736
3123437
3265046
3349227
3467324
3546041
3680754
3793239
3816056
3941029
4012238
4183011
4239700
4330289
4488490
4516559
4631184
4797693
4862726
4949883
5011148
5169641
5246018
5329415
5430184
5539317
5661502
5751955
5896772
5988449
6088282
6136863
6227232
6375149
6464470
6500619
6640124
6705145
6856850
6973239
7085752
7171333
7244590
7330307
7418740
7580849
7611914
7709039
7806224
7985405
8095270
8169563
8261388
8321513
8461666
8500647
8676648
8721621
8851134
8983891
9099044
9104673
9272570
9328191
9454848
9527245
9687798
9775915
9845820
9928921
10042290
10107063
10231672
10378853
10420782
10524611
10628084
10735153
10840938
10987567
11009680
11140413
11267078
11347259
11410828
11568777
11629442
11730119
11895432
11944085
12024094
12151411
12290468
12313697
12444346
12517631
12672672
12779277
12894902
12958635
13066652
13118201
13249042
13374263
13409240
13551589
13694382
13771107
13813204
13972497
14045770
14159727
14208400
14351901
14470854
14538907
14664556
14798281
14872290
14998887
15010248
15177109
15208606
15322995
15413892
15522657
15693914
15705151
15824960
15961005
16094966
16126187
16212316
16362201
16445618
16541047
16661400
16742341
16851918
16934275
17014324
17144209
17236810
17385167
17489456
17557501
17615910
17776091
17871468
17968905
18038882
18172775
18215976
18388597
18482206
18510067
18606916
18743425
18886362
18942783
19025248
19183341
19230134
19398923
19421852
19589465
19626258
19740151
19816728
19966821
20045582
20156131
20233716
20357297
20482890
20524655
20660144
20741405
20860870
20984283
21093452
21162505
21201730
21369671
21476776
21593013
21622686
21717939
21852836
21978145
22026970
22150975
22290944
22387405
22475510
22556491
22696348
22767865
22809042
22923671
23025912
23105541
23227822
23354339
23435700
23558097
23676682
23707375
23831632
23901661
24097318
24131899
24287148
24349321
24440834
24508583
24644328
24753269
24806206
24943923
25064292
25133697
25279450
25383999
25410624
25579117
25639158
25739723
25815292
25937305
26003186
26128471
26208920
26352869
26449454
26540451
26646644
26764977
26869290
26914575
27018960
27184765
27282502
27330299
27438540
27538921
27661922
27776935
27874184
27995189
28092830
28146515
28297732
28307233
28482298
28599135
28642144
28710957
28894806
28997803
29003868
29177017
29209042
29308887
29422296
29500677
29628846
29752611
29871316
29935633
30002954
30161327
30257392
30300125
30473926
30568379
30628364
30780201
30843618
30946919
31058472
31193045
31240990
31343507
31489892
31500641
31614202
31703839
31814528
31900205
32049142
32141099
32214716
32321913
32485394
32554967
32677400
32738341
32899086
32904611
33060180
33183441
33227402
33382063
33435120
33521949
33627462
33727035
33815756
33910441
34090274
34142791
34293192
34351317
34409374
34511987
34621316
34771489
34879578
34908575
35026080
35194349
35225462
35339275
35464412
35578937
35631218
35757879
35854872
35969093
36077166
36187779
36242100
36361105
36499850
36545039
36627856
36754653
36866534
36948347
37089196
37173545
37246434
37306279
37410920
37571989
37689630
37790227
37885540
37900321
38039386
38173983
38246272
38338381
38412726
38588587
38646044
38794361
38803922
38977207
39006520
39126373
39245742
39320035
39424340
39509713
39619594
39754223
39898832
39967453
40012934
40182747
40231916
40301801
40498722
40514823
40631752
40754549
40855422
40992147
41054500
41139713
41296666
41312767
41481152
41574797
41632854
41795531
41816636
41937433
42052050
42124599
42253432
42375717
42436782
42511299
42643508
42709681
42809770
42998351
43080560
43176125
43221414
43325499
43456492
43583177
43657602
43763335
43812360
43928693
44080670
44141907
44262180
44381569
44478298
44589695
44624288
44759949
44857078
44925899
45077020
45182137
45231634
45361879
45471512
45545573
45698862
45716323
45830964
45964401
46084650
46150927
46276720
46377597
46463334
46580219
46602956
46712265
46808322
46915143
47013288
47172565
47204958
47390259
47437316
47589825
47653242
47770655
47838528
47914989
48073238
48139371
48257372
48325433
48473490
48553879
48605944
48761893
48846222
48965507
49080116
49165649
49233290
49386575
49443920
49566301
49643014
49795355
49807980
49904745
50043618
50101991
50298952
50343573
50400830
50547251
50637988
50767745
50863354
50906303
51041536
51123981
51251190
51354923
51449916
51536313
51615922
51799255
51889432
51931781
52009710
52115843
52230068
52344529
52452234
52515631
52626288
52750525
52863142
52911355
53002988
53182921
53231010
53324039
53403048
53531221
53673790
53700467
53817028
53957921
54043514
54199359
54203904
54319437
54444694
54501771
54608924
54706745
54872018
54916023
55058552
55176197
55285838
55334563
55431028
55571473
55612810
55765903
55888176
55936061
56010310
56162555
56230508
56308425
56445506
56582567
56697736
56771573
56895006
56913139
57056612
57107681
57201498
57370015
57416448
57596781
57622230
57735723
57899772
57917081
58046002
58172375
58202808
58313605
58487726
58531843
58646196
58721649
58877482
58900911
59030512
59126205
59220422
59342587
59413356
59563977
59620546
59747271
59824456
59981845
60035902
60142387
60233668
60338273
60403514
60590527
60641152
60767181
60803478
60955531
61000828
61170649
61233234
61355863
61499224
61576325
61677806
61760803
61833524
61960497
62016202
62101647
62225392
62354461
62495046
62555323
62685452
62706217
62813410
62956263
63028520
63193109
63249918
63371923
63487172
63508385
63674202
63717919
63825632
63953773
64022838
64170187
64285212
64342809
64436562
64538807
64645048
64777829
64867822
64935747
65029364
65141841
65273706
65317519
65430544
65535485
65668710
65798523
65859116
65912873
66072514
66132423
66259112
66333941
66490750
66561107
66618148
66766977
66878394
66961279
67055968
67179085
67214966
67331627
67471740
67548249
67633618
67710999
67857752
67978149
68086126
68195683
68283060
68358833
68402218
68549999
68642160
68781437
68892550
68978299
69092364
69151945
69288354
69399751
69401992
69546773
69694110
69730483
69873316
69951393
70055706
70147167
70203872
70344909
70480182
70502027
70657628
70790041
70893170
70914295
71058680
71102117
71243086
71324451
71455060
71576657
71628746
71718031
71825712
71918333
72067014
72145595
72221612
72396169
72424322
72534311
72630440
72755189
72895966
72976243
73010500
73138305
73236890
73370559
73412224
73517837
73651766
73786507
73802140
73961529
74075122
74142615
74205752
74372069
74449326
74597891
74667860
74783473
74871434
74976271
75034192
75186045
75274566
75327003
75412620
75541257
75653186
75780583
75854312
75996629
76014302
76121171
76216868
76360065
76411130
76594847
76644128
76761965
76879606
76916779
77026588
77177145
77246866
77340055
77477112
77556901
77681134
77769379
77806004
77955825
78055914
78113199
78257168
78389245
78481574
78575707
78658604
78754953
78871554
78984167
79003688
79138741
79283710
79381939
79457988
79588353
79624154
79751839
79882912
79987949
80079254
80167499
80247036
80341593
80434226
80537367
80622072
80773957
80847758
80935235
81083380
81176881
81266250
81336655
81471728
81595517
81620070
81755131
81849420
81960681
82099874
82159079
82212520
82313973
82471742
82550867
82645124
82783233
82847418
82990815
83064832
83154893
83215574
83390859
83492540
83564313
83670514
83715479
83842872
83903493
84091406
84151075
84257684
84334545
84493994
84555599
84607888
84704541
84871398
84971643
85002380
85188489
85284322
85304103
85418632
85568629
85611614
85781459
85800644
85911393
86092570
86180415
86264608
86368461
86470038
86531051
86603260
86777721
86896530
86991031
87045624
87174213
87203598
87350051
87466900
87539697
87654538
87780111
87883184
87978909
88033510
88141851
88264140
88389513
88411266
88502791
88696424
88762869
88836798
88935571
89059908
89127905
89213338
89323103
89442592
89599117
89689782
89726155
89865052
89986265
90011506
90198135
90273016
90345029
90408814
90524611
90692820
90775249
90889002
90972335
91051728
91162365
91270022
91325307
91443404
91505161
91658210
91774407
91840520
91938005
92079134
92152595
92269380
92327169
92403642
92573407
92630304
92780237
92822198
92924491
93025820
93133721
93238514
93306615
93438904
93572357
93680846
93710147
93897524
93930737
94078698
94179183
94238800
94379805
94461158
94508155
94643628
94797929
94828386
94988391
95033064
95165557
95254366
95398515
95404868
95575617
95615834
95754271
95812192
95960941
96080342
96150027
96211868
96319545
96459058
96587415
96626424
96710117
96826446
96935939
97086900
97116785
97296074
97315983
97438960
97502685
97669158
97703131
97830316
97904233
98014274
98117191
98292776
98314549
98460254
98598611
98620580
98762657
98809754
98989759
99060224
99169613
99242934
99304267
99495196
99598873
99678482
99796247
99889880
99942437
