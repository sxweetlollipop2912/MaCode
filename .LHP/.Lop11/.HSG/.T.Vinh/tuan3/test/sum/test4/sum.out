380830610223846
